qa/workunits/ceph-disk/ceph-disk-no-lockbox

```diff
@@ -2310,7 +2310,7 @@ class PrepareData(object):
         if self.args.osd_uuid is None:
             self.args.osd_uuid = str(uuid.uuid4())
 
-    def prepare_device(self, *to_prepare_list):
+    def prepare_device(self):
```

Comment:
yes, technically, this parameter is not used at all. but IMHO, `PrepareData.prepare_device()` is not just a utility method used by its children classes. it also act as an interface documenting what a derived class should implement.